test/e2e/mcd_test.go

```diff
@@ -745,7 +745,7 @@ func TestDontDeleteRPMFiles(t *testing.T) {
 func TestIgn3Cfg(t *testing.T) {
 	cs := framework.NewClientSet("")
 
-	delete := helpers.CreateMCP(t, cs, "infra")
+	deleteFunc := helpers.CreateMCP(t, cs, "infra")
```

Comment:
Note to reviewer: I changed this because `delete()` is a built-in function in Golang. While it was not causing any problems, I opted to proactively fix it.